mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -344,6 +344,10 @@ final class OneVsRest @Since("1.4.0") (
       multiclassLabeled.unpersist()
     }
 
+    val instrLog = Instrumentation.create(this, multiclassLabeled)
```

Comment:
i think `multiclassLabeled` is the dataset we want, but i'm not sure what it's supposed to be used for in instrumentation.scala so i could be wrong